packages/lit-dev-content/site/tutorials/content/svg-templates/03.md

```diff
@@ -0,0 +1,168 @@
+This section of the tutorial covers the basics of `<clipPath>` and
+`<g>` elements in SVG including how to:
+
+- Create a clip path
+- Create a group of groups with `<use>`
+- Apply a clip path to a group
+
+### Learn
+
+In pattern making, a _tile_ is a polygon that can be repeated across
+a two dimensional plane. The tile in this demo will be a series of
+transformed texts.
+
+Clip paths are polygons used to restrict where elements are "painted".
+In SVG, a clip path is defined with the [`<clipPath>`](https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path)
+element.
+
+In the example below, a clip path will be created in the shape of a 
+rectangle with an `id` of `rect-clip`.
+
+```ts
+const helloClipPath = svg`
+	<clipPath>
+		<rect width="200" height="200"></rect>
+	</clipPath>
+`;
+```
+
+In order to apply a clip path to an SVG element, the `clip-path`
+property should reference a `<clipPath>` element by an `id`. The
+`<rect>` below will be clipped by the clip-path above by referencing
+`#react-clip`.
+
+```ts
+const helloTile = svg`
+	<rect
+		clip-path="url(#rect-clip)"
+		width="300"
+		height="300"
+		fill="#000">
+	</rect>
+```
+
+### Apply
+
+Create a clip path composed of a 200px square. This `<clipPath>` will
+restrict paints to create a _tile_. Give the clip-path an `id` called `rect-clip`.
+
+```ts
+const createClipPath = () => svg`
```

Comment:
Sorry this is late in revew, but why don't you refer to this as its concept like `createMotif` and `createElement`? Why not `createTileBoundary` or something of the like?

Reply:
lets do that!

Reply:
done!